tests/playbooks/mp/test_transport_node_profiles.yml

```diff
@@ -19,7 +19,6 @@
         host_switch_spec:
           resource_type: StandardHostSwitchSpec
           host_switches: "{{item.host_switches}}"
-          transport_zone_endpoints: "{{item.transport_zone_endpoints}}"
```

Comment:
Bug Fixed - 2619365 - nsxt_transport_node_profiles.py fails if transport_zone_endpiont is not provided outside of host_switches